lib/utils/cli.go

```diff
@@ -522,3 +523,21 @@ type PredicateError struct {
 func (p PredicateError) Error() string {
 	return fmt.Sprintf("%s\nCheck syntax at https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering", p.Err.Error())
 }
+
+func FormatAlert(alert types.ClusterAlert) string {
+	var buf bytes.Buffer
+	if runtime.GOOS == constants.WindowsOS {
```

Comment:
Nit: Switch would work better both here, and below too (`switch alert.Spec.Severity`).